server/models/Profile.js

```diff
@@ -28,12 +28,17 @@ const profileSchema = new mongoose.Schema({
   },
   birthday: {
     type: Date,
-    default: null
+    default: null,
   },
   photo: {
     type: String,
     default: "",
   },
+  accountType: {
+    type: String,
+    enum: ["pet_sitter", "pet_owner"],
+    default: "pet_owner",
```

Comment:
Please use `camelCase` for naming these fields.

Reply:
I need this value to filter menu items in Navbar at FE. There already defined as snake_case, if I use camelCase here, I need to write extra code to convert it. That is why I use snake_case. And I hope to keep snake_case here.